OWNERS_ALIASES

```diff
@@ -467,14 +441,9 @@ aliases:
     - msau42
     - jsafrane
     - xing-yang
-  sig-security-approvers:
-    - IanColdwater
-    - tabbysable
-  sig-security-reviewers:
-    - IanColdwater
-    - tabbysable
+  sig-security-approvers: []
```

Comment:
Looks like sig-security does not own any code in k/k!

Reply:
should these aliases be dropped instead of emptied then?